2-datatypes/boolean.js

```diff
@@ -0,0 +1,16 @@
+const prompt = require("prompt");
+
+prompt.start();
+
+prompt.get(["input"], main)
+function main(err, res) {
+    let parsedInput = false;
+    if (res.input === "true") {
+        parsedInput === true;
```

Comment:
This is not assignment

Reply:
sorry
sir I am blocked 😊

can you give me some more hints.